server/routes/user.js

```diff
@@ -14,4 +14,6 @@ router.get('/:username/followers', protect, followers);
 
 router.get('/:username/followings', protect, followings);
 
+router.get('/:username/followSugestions', protect, followSugestions);
```

Comment:
when were these /followings routes merged in? I can't seem to find the PR for it

Reply:
All of these routers for `/:username` routes are not required to include `:username` because it's always the logged in users. in which should be accessible by the server based on the cookie that is passed through